src/main/java/baritone/process/MineProcess.java

```diff
@@ -85,7 +85,7 @@ public PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
             if (curr >= desiredQuantity) {
                 logDirect("Have " + curr + " " + item.getItemStackDisplayName(new ItemStack(item, 1)));
                 cancel();
-                return null;
+                return new PathingCommand(null, PathingCommandType.REQUEST_PAUSE);
```

Comment:
why?

Reply:
because else it trows and crashes the game. this was annoying because I thaught the crashing was my falt but turns out that MineProcess.java have that one difference compared to all other processes.

Reply:
i mean the implied question is why didn't this used to crash, and why does it only crash with your other changes applied

Reply:
it crashes when its called twice in one frame.